aten/src/THNN/generic/VolumetricConvolutionMM.c

```diff
@@ -132,68 +134,89 @@ static void THNN_(unfolded_acc_vol)(
           long outputWidth,
           long outputHeight)
 {
-  long nip;
   real *input_data = THTensor_(data)(input);
   real *finput_data = THTensor_(data)(finput);
+  int in_omp = omp_in_parallel();
```

Comment:
this and other references to `omp_*` C functions in this file need a `#ifdef _OPENMP` or equivalent guard